.github/workflows/rust.yml

```
@@ -80,6 +89,15 @@ jobs:
               with:
                 node-version: 22
 
+            - name: Cache
+              uses: actions/cache@v4
+              with:
+                path: |
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)